code/game/atoms_movable.dm

```diff
@@ -1182,3 +1188,8 @@
 */
 /atom/movable/proc/keybind_face_direction(direction)
 	setDir(direction)
+
+/atom/movable/proc/update_sounds()
+	if(length(attachedSounds))
+		for (var/sound/S as anything in attachedSounds)
```

Comment:
better var name, please

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names